geode-cq/src/test/java/org/apache/geode/management/internal/cli/commands/ListClientCommandDUnitTest.java

```diff
@@ -1,402 +1,159 @@
+      assertNotNull(dataRegion);
+      dataRegion.put("k1", "v1");
+      dataRegion.put("k2", "v2");
+    }, client1, client2);
+
+    locator.waitTillClientsAreReadyOnServers("server-1", server1port, 2);
+
+    CommandResult result =
+        gfsh.executeAndAssertThat("list clients").statusIsSuccess().getCommandResult();
+
+    List<String> clientList =
+        result.getColumnFromTableContent("Client Name / ID", "section1", "TableForClientList");
+    assertThat(clientList).hasSize(2);
+    try {
+      assertThat(clientList.get(0)).contains("client-1");
+      assertThat(clientList.get(1)).contains("client-2");
+    } catch (AssertionError e) {
+      assertThat(clientList.get(0)).contains("client-2");
+      assertThat(clientList.get(1)).contains("client-1");
     }
 
-    for (String server : serverNames) {
-      assertTrue(server.contains(serverName1) || server.contains(serverName2));
-    }
+    assertThat(
+        result.getColumnFromTableContent("Server Name / ID", "section1", "TableForClientList"))
+            .hasSize(2)
+            .containsExactlyInAnyOrder("member=server-1,port=" + server1port,
+                "member=server-1,port=" + server1port);
 
-    ClientCommandsTestUtils.closeNonDurableClient(Host.getHost(0).getVM(2));
-    ClientCommandsTestUtils.closeCacheServer(Host.getHost(0).getVM(1));
-    ClientCommandsTestUtils.closeCacheServer(Host.getHost(0).getVM(3));
+    // shutdown the clients
+    cluster.stopMember(3);
+    cluster.stopMember(4);
   }
 
-  private void setupSystem() throws Exception {
-    disconnectAllFromDS();
-    setUpJmxManagerOnVm0ThenConnect(ClientCommandsTestUtils.getServerProperties());
-
-    final VM manager = Host.getHost(0).getVM(0);
-    final VM server1 = Host.getHost(0).getVM(1);
-    final VM client1 = Host.getHost(0).getVM(2);
-    final VM server2 = Host.getHost(0).getVM(3);
-
-    port0 = startCacheServer(server1, regionName);
-    port1 = startCacheServer(server2, regionName);
-    startNonDurableClient(client1, server1, port0);
-    startNonDurableClient(client1, server2, port1);
-
-    waitForListClientMbean3();
-
-    manager.invoke("get client Id", () -> {
-      Cache cache = GemFireCacheImpl.getInstance();
-      SystemManagementService service =
-          (SystemManagementService) ManagementService.getExistingManagementService(cache);
-      DistributedMember serverMember = ClientCommandsTestUtils.getMember(server1);
-      final ObjectName cacheServerMBeanName = service.getCacheServerMBeanName(port0, serverMember);
-      CacheServerMXBean bean = service.getMBeanProxy(cacheServerMBeanName, CacheServerMXBean.class);
-      return bean.getClientIds()[0];
+  @Test
+  public void oneClientConnectToTwoServers() throws Exception {
+    int server1port = server1.getPort();
+    int server2port = server2.getPort();
+    Properties client1props = new Properties();
+    client1props.setProperty("name", "client-1");
+    client1 = cluster.startClientVM(3, client1props, cf -> {
```

Comment:
I'm a fan of extracting ID numbers like `3` here into a local variable i.e. client1ID, so when we use it later its obvious what we are working with next time we use it (shutting down the client).